perun-base/src/test/resources/test-schema.sql

```diff
@@ -1552,6 +1552,84 @@ create table groups_to_register (
 						constraint grpreg_group_fk foreign key (group_id) references groups(id) on delete cascade
 );
 
+-- CONSENTS_DEFINITIONS - consents definitions for facilities, services and attributes
+create table consents_definitions (
+	id integer not null,
+	name varchar not null,
+	text varchar,
+	created_at timestamp default statement_timestamp() not null,
+	created_by varchar default user not null,
+	modified_at timestamp default statement_timestamp() not null,
+	modified_by varchar default user not null,
+	created_by_uid integer,
+	modified_by_uid integer,
+	constraint con_def_pk primary key(id),
+	constraint con_def_u unique(name)
+);
+
+-- CONSENTS - relation between users and consent definitions
+create table consents (
```

Comment:
One last thing - we might want to have a column for ID in this table too. I am not sure if it is necessary, but I assume the Consent object will extend the PerunBean object, which has an ID attribute. What do you think?